From ecfe16ba57601c3cdbbe530e83a4e10f08f08e35 Mon Sep 17 00:00:00 2001
From: Chin-Ting Kuo <chin-ting_kuo@aspeedtech.com>
Date: Thu, 24 Jul 2025 09:53:35 +0800
Subject: [PATCH 03/32] arm: aspeed: Add AST2700-A2 chip ID

Update chip ID table for AST2700-A2.

Signed-off-by: Chin-Ting Kuo <chin-ting_kuo@aspeedtech.com>
Change-Id: I56f3791aa2e6524f481346f6877627c468f28a5a
(cherry picked from commit 5e4644e0174a070009ba8ebed11708ea60ac67c4)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 arch/arm/mach-aspeed/ast2700/cpu-info.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/mach-aspeed/ast2700/cpu-info.c b/arch/arm/mach-aspeed/ast2700/cpu-info.c
index 81351ae7a4..9d56284fc0 100644
--- a/arch/arm/mach-aspeed/ast2700/cpu-info.c
+++ b/arch/arm/mach-aspeed/ast2700/cpu-info.c
@@ -24,6 +24,9 @@ static struct soc_id soc_map_table[] = {
 	SOC_ID("AST2750-A1", 0x0601000306010003),
 	SOC_ID("AST2700-A1", 0x0601010306010103),
 	SOC_ID("AST2720-A1", 0x0601020306010203),
+	SOC_ID("AST2750-A2", 0x0602000306020003),
+	SOC_ID("AST2700-A2", 0x0602010306020103),
+	SOC_ID("AST2720-A2", 0x0602020306020203),
 };
 
 void ast2700_print_soc_id(void)
-- 
2.34.1

